src/test/functional/features/joint-final-order.feature

```diff
@@ -43,3 +41,71 @@ Feature: Joint final order
     When I click "Submit"
     Then the page URL should be "/applicant2/hub-page"
     Then the page should include "You and your wife have both confirmed you want to finalise the divorce."
+
+  @nightly
+  Scenario: [2] Applicant 1 joint switch to sole final order journey
+    Given I login with applicant "1"
+    Then the page should include "You can now apply for a ‘final order’."
+
+    When I click "Apply for final order"
+    Then the page should include "Do you want to finalise your divorce?"
+
+    Given I select "I want to finalise my divorce"
+    When I click "Submit"
+    Then the page URL should be "/hub-page"
+    And the page should include "You have applied for a ‘final order’. Your husband also has to apply because this is a joint application."
+    And the page should include "They have been sent an email reminder."
+
+    Then I set the case state to "AwaitingJointFinalOrder"
+    And a superuser updates "dateFinalOrderSubmitted" with "2022-09-21T08:08:34.548"
+
+    Given I click "Sign out"
+    When I login with applicant "1"
+    Then the page URL should be "/hub-page"
+    Then the page should include "If you do not think they will confirm the application then you can finalise your divorce"
+
+    Given I click "as a sole applicant"
+    Then the page URL should be "/how-to-finalise"
+    And the page should include "How to finalise your divorce"
+
+    Given I click "Confirm"
+    Then the page URL should be "/how-to-finalise"
+    And the page should include "There was a problem"
```

Comment:
do we need to test mandatory fields here? I think there are already tests for that

Reply:
Should be tested as part of common components, test removed.

Reply:
fixed